joi/index.js

```diff
@@ -14,6 +14,12 @@ const joiSimpleDate = () => {
 	return Joi.string().regex(/^\d{4}-(0[1-9]|1[012])-(0[1-9]|[12][0-9]|3[01])$/);
 };
 
+const joiStateAbbreviation = () => {
+	return Joi.string().regex(
+		/^(?:(A[KLRZ]|C[AOT]|D[CE]|FL|GA|HI|I[ADLN]|K[SY]|LA|M[ADEINOST]|N[CDEHJMVY]|O[HKR]|P[A]|RI|S[CD]|T[NX]|UT|V[AT]|W[AIVY]))$/
```

Comment:
what's your thinking behind using a capturing group wrapped in a non-capturing group?

Reply:
Ah didn't even notice the nested capturing group. I'll remove. Totally copied this from somewhere.